modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/index/hash/PageMemoryHashIndexStorage.java

```diff
@@ -91,12 +91,15 @@ public class PageMemoryHashIndexStorage implements HashIndexStorage {
      * @param freeList Free list to store index columns.
      * @param hashIndexTree Hash index tree instance.
      * @param indexMetaTree Index meta tree instance.
+     * @param lastBuiltIndexRowUuid Last row ID uuid that has been processed by an ongoing index build process, {@code null} if the process
+     *      has finished.
      */
     public PageMemoryHashIndexStorage(
             HashIndexDescriptor descriptor,
             IndexColumnsFreeList freeList,
             HashIndexTree hashIndexTree,
-            IndexMetaTree indexMetaTree
+            IndexMetaTree indexMetaTree,
+            @Nullable UUID lastBuiltIndexRowUuid
```

Comment:
I recommended passing meta object, not the id itself, can you please change that?
Meta is an object, that has everything grouped, and passing such object into a constructor is a good thing. Passing some arbitrary uuid and, potentially, the rest of meta fields, looks wrong

Reply:
Fix it.